repos/spack_repo/builtin/packages/libfabric/package.py

```diff
@@ -138,6 +140,7 @@ class Libfabric(AutotoolsPackage, CudaPackage):
     depends_on("cassini-headers", when="fabrics=cxi")
     depends_on("cxi-driver", when="fabrics=cxi")
     depends_on("xpmem", when="fabrics=xpmem")
+    depends_on("gdrcopy", when="+gdrcopy")
```

Comment:
Unrelated to your change, but I just noticing that we added xpmem as a fabric, that's not right.

Reply:
Sorry, wasn't suggesting we change in this PR.  Was just noting it for posterity.